google-datacatalog-connectors-commons/src/google/datacatalog_connectors/commons/datacatalog_facade.py

```diff
@@ -123,23 +138,95 @@ def __entry_was_updated(cls, current_entry, new_entry):
             new_update_time != 0 and current_update_time != new_update_time
 
         return updated_time_changed or not cls.__entries_are_equal(
-            current_entry, new_entry)
+            current_entry=current_entry, new_entry=new_entry)
 
     @classmethod
-    def __entries_are_equal(cls, entry_1, entry_2):
+    def __entries_are_equal(cls, *, current_entry, new_entry):
+        """
+        Compares if two entries are equal.
+        Since the args order must be respected, this method uses
+        named args.
+
+        :param current_entry: An Entry object that is
+        synchronized in Data Catalog.
+        :param new_entry: An Entry object not yet
+        synchronized in Data Catalog.
+        """
         object_1 = utils.ValuesComparableObject()
-        object_1.user_specified_system = entry_1.user_specified_system
-        object_1.user_specified_type = entry_1.user_specified_type
-        object_1.display_name = entry_1.display_name
-        object_1.description = entry_1.description
-        object_1.linked_resource = entry_1.linked_resource
+        object_1.user_specified_system = current_entry.user_specified_system
+        object_1.user_specified_type = current_entry.user_specified_type
+        object_1.display_name = current_entry.display_name
+        object_1.description = current_entry.description
+        object_1.linked_resource = current_entry.linked_resource
 
         object_2 = utils.ValuesComparableObject()
-        object_2.user_specified_system = entry_2.user_specified_system
-        object_2.user_specified_type = entry_2.user_specified_type
-        object_2.display_name = entry_2.display_name
-        object_2.description = entry_2.description
-        object_2.linked_resource = entry_2.linked_resource
+        object_2.user_specified_system = new_entry.user_specified_system
+        object_2.user_specified_type = new_entry.user_specified_type
+        object_2.display_name = new_entry.display_name
+        object_2.description = new_entry.description
+        object_2.linked_resource = new_entry.linked_resource
+
+        return object_1 == object_2 and cls.__schemas_are_equal(
+            current_schema=current_entry.schema, new_schema=new_entry.schema)
+
+    @classmethod
+    def __schemas_are_equal(cls, *, current_schema, new_schema):
+        current_columns = current_schema.columns
+        new_columns = new_schema.columns
+
+        columns_are_equal = set([c_1.column for c_1 in current_columns]) == \
```

Comment:
The names `c_1` and  `c_2` are not clear in this context. WDYT about renaming these variables?